app/src/pages/authoring/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { Button, Center, Paper, SegmentedControl, Stack, Title, createStyles } from '@mantine/core';
+import { v4 as uuidv4 } from 'uuid';
+import { useState } from 'react';
+import { trpc } from '../../util/trpc';
+import { MeasureSkeleton, LibrarySkeleton } from '@/util/authoringFixtures';
+import { useRouter } from 'next/router';
+import { notifications } from '@mantine/notifications';
+import { AlertCircle, CircleCheck } from 'tabler-icons-react';
+import { ArtifactResourceType } from '@/util/types/fhir';
+
+const useStyles = createStyles(theme => ({
+  centeredContainer: {
+    height: '100%'
+  }
+}));
+
+export default function AuthoringPage() {
+  const [resourceType, setResourceType] = useState<ArtifactResourceType>('Measure');
+
+  const ctx = trpc.useContext();
+  const { classes } = useStyles();
+
+  const draftMutation = trpc.draft.createDraft.useMutation({
```

Comment:
Overall question here, do we want to immediately create a new Measure or Library when we click create? Or do we want to hold off until the user clicks submit?

Reply:
Unsure for now. Maybe it will become clearer which we should do as we get some more inputs in there for things you can add to the artifact. For now it's probably fine as is